src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let rentalCost = 0;
+
+  rentalCost = days * 40;
+
+  if (days >= 3) {
+    rentalCost = (days * 40) - 20;
```

Comment:
it better to return value from this block, not reassign variable 
[Checklist](https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md) p2,3